docs/src/univariate.md

```diff
@@ -327,6 +327,13 @@ LogNormal
 plotdensity((0, 5), LogNormal, (0, 1)) # hide
 ```
 
+```@docs
+LogUniform
+```
+```@example plotdensity
+plotdensity((0, 11), LogUniform, (1, 10)) # hide
```

Comment:
Did you see that the plot looks ok? I think we are done.

Reply:
![image](https://user-images.githubusercontent.com/26102/141289896-5091a7a7-7797-4894-bce8-658f78ec0558.png)

Reply:
![image](https://user-images.githubusercontent.com/26102/141289929-77ccfce1-d06a-480c-8d45-292980ec383f.png)

Reply:
It's copied from the docs that I built locally.